yellowbox/extras/azure_storage.py

```diff
@@ -33,6 +34,8 @@ def __init__(self, docker_client: DockerClient, image: str = "mcr.microsoft.com/
         container = create_and_pull(
             docker_client, image, "azurite-blob --blobHost 0.0.0.0", publish_all_ports=True)
         super().__init__(container, **kwargs)
+        self.account_name = DEFAULT_ACCOUNT_NAME
+        self.account_key = DEFAULT_ACCOUNT_KEY
```

Comment:
add parameters to __init__? if these attributes are needed before `start`, `run` will be unusable

Reply:
It's not a parameter. It's attributes. Is it better now?